src/Firehose.Web/Authors/Telerik.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Firehose.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.ServiceModel.Syndication;
+
+namespace Firehose.Web.Authors
+{
+	public class Telerik : IAmACommunityMember, IFilterMyBlogPosts
+	{
+		public string FirstName => "Telerik";
+
+		public string LastName => string.Empty;
+
+		public string StateOrRegion => "Global";
+
+		public string EmailAddress => "telerikxamarinteam@progress.com";
+
+		public string ShortBioOrTagLine => "Progress Telerik UI libraries equip .NET ninjas with a full arsenal of weapons to help you create beautiful, modern and future-proof applications quickly and intuitively.";
+
+		public Uri WebSite => new Uri("https://www.telerik.com/");
+
+		public string TwitterHandle => "telerik";
+
+		public string GitHubHandle => "telerik";
+
+		public string GravatarHash => string.Empty;
+
+		public IEnumerable<Uri> FeedUris
+		{
+			get
+			{
+				yield return new Uri("https://feeds.telerik.com/blogs/");
+			}
+		}
+
+		// Bedford, MA Headquarters.
+		public GeoPosition Position => new GeoPosition(42.512778, -71.251500); 
+
+		public string FeedLanguageCode => "en";
+
+		// Categories are not suffient enough for filer. Identifying 'xamarin' in the Title will be sufficient.
+		public bool Filter(SyndicationItem item)
+		{
+			return item.Title.Text.ToLowerInvariant().Contains("xamarin");
```

Comment:
We already filter Xamarin in title and category. So please go ahead and remove the `IFilterMyBlogPosts` interface and method.